activestorage/lib/active_storage/attached/model.rb

```diff
@@ -61,7 +61,7 @@ def #{name}
 
           def #{name}=(attachable)
             attachment_changes["#{name}"] =
-              if attachable.nil?
+              if attachable.nil? || attachable == ""
```

Comment:
:thinking: Any reason to not use `blank?`?

Reply:
I simply wanted to add support for empty strings, since that’s how HTML form fields work when they have no `value`. The previous code checked for `nil?` instead of being falsely, which I assumed was intentional.

I feel like whitespace-only strings and `false` should still raise an `ActiveSupport::MessageVerifier::InvalidSignature: mismatched digest`.

Additionally, `blank?` is about 2.17x slower than what I've got here, when given a signed id:

```ruby
# frozen_string_literal: true

require 'benchmark/ips'
require 'active_support'

str = +'BAh7CEkiCGdpZAY6BkVUSSIbZ2lkOi8vZHdlbGwtd2ViL1VzZXIvMQY7AFRJIgxwdXJwb3NlBjsAVEkiDGRlZmF1bHQGOwBUSSIPZXhwaXJlc19hdAY7AFRJIh0yMDIzLTA3LTAyVDEzOjUwOjE2LjM4MVoGOwBU--4335d744fdc394d750210728675cb8ef9e7e0ef0'

Benchmark.ips do |x|
  x.report('==') { str.nil? || str == '' }
  x.report('blank?') { str.blank? }

  x.compare!
end

Warming up --------------------------------------
                  ==     1.750M i/100ms
              blank?   802.528k i/100ms
Calculating -------------------------------------
                  ==     17.498M (± 0.5%) i/s -     87.495M in   5.000509s
              blank?      8.052M (± 0.9%) i/s -     40.929M in   5.083819s

Comparison:
                  ==: 17497713.1 i/s
              blank?:  8051557.4 i/s - 2.17x  slower
```

`blank?` would solve Jean's concerns of mixing HTML form concerns in the model layer, but it would come at the expense of allowing more values to not raise an error (which feels incorrect), and being slower.